JS/Lesson24/saffet/filter.js

```diff
@@ -345,6 +378,14 @@ const studentsExercise9 = [
 	},
 ];
 
+const lowerThenC = studentsExercise9.filter(student => {
+	if(student.grade === "D+" || student.grade === "D"){
```

Comment:
```suggestion
	if(student.grade !== "D+" || student.grade !== "D"){
```

It would be good to also add C-, D-, F+, F, F-